README.md

```diff
@@ -8,30 +8,35 @@ in the [documentation](doc/DOCUMENTATION.md).
 Installation via [Opam](http://opam.ocaml.org/)
 ---------------------
 
+We will only publish a new version of `lambdapi` Opam pacakge when the
+development has reached a more stable test. For now, we advise you to
+pin the development repository to get the latest bug fixes.
+
 ```bash
-opam install lambdapi
+opam pin add lambdapi https://github.com/Deducteam/lambdapi.git
```

Comment:
It doesn't look like a good idea to pin add the development version. Please revert.

Reply:
It is worse for a user to install a completely outdated version. We will change that when we do a stable release, which I guess should be done soon.

Reply:
Right. My point was to anticipate this new release.